### DISCUSSION TITLE
Welcome to ui5-webcomponents Discussions!

### DISCUSSION THREAD

**Discussion by ilhan007** [UI5/webcomponents#2554] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and be open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by hrstoyanov** [comment]
Thank you @ilhan007 !
